tests/ruby-sass/media-width.scss

```diff
@@ -0,0 +1,37 @@
+@import "../setup";
+
+.media-width-from {
+  @include wtr-media-width(from, 250px) {
+    color: red
+  }
+}
+
+.media-width-until {
+  @include wtr-media-width(until, 250px) {
+    color: blue
+  }
+}
+
+.media-width-equal {
+  @include wtr-media-width(equal, 250px) {
+    color: lime
+  }
+}
+
+.media-width-between {
+  @include wtr-media-width(between, 250px, 450px) {
+    color: purple
```

Comment:
Color `purple` should be written in hexadecimal form as `#800080`<br>Declaration should be terminated by a semicolon